lib/at_cmd/at_cmd.c

```diff
@@ -308,10 +323,9 @@ void at_cmd_set_notification_handler(at_cmd_handler_t handler)
 
 static int at_cmd_driver_init(struct device *dev)
 {
-	static bool initialized;
-
-	if (initialized) {
-		return 0;
+	if (atomic_test_bit(&running, 0)) {
```

Comment:
Do we need this change? `at_cmd` does not need to be re-initialized anymore. Please keep the diff minium, avoid unnecessary changes.

Reply:
Removed this change